unskript-ctl/unskript-client.py

```diff
@@ -567,14 +600,20 @@ def create_jit_runbook(check_list: list):
        :rtype: None
     """
     nb = nbformat.v4.new_notebook()
-    failed_notebook = os.environ.get('EXECUTION_DIR', '/unskript/data').strip('"') + '/workspace/' + str(uuid.uuid4()) + '.ipynb'
+    if os.path.exists(os.environ.get('EXECUTION_DIR') + '/workspace') is False:
+        os.makedirs(os.environ.get('EXECUTION_DIR') + '/workspace')
+
+    failed_notebook = os.environ.get('EXECUTION_DIR', '/unskript/data/execution').strip('"') + '/workspace/' + str(uuid.uuid4()) + '.ipynb'
     for check in check_list:
         s_connector = check.get('metadata').get('action_type')
         s_connector = s_connector.replace('LEGO', 'CONNECTOR')
         cred_name, cred_id = get_creds_by_connector(s_connector)
         # No point proceeding further if the Credential is incomplete
         if cred_name is None or cred_id is None:
-            print('\x1B[1;20;46m' + f"~~ Skipping {check.get('name')} {cred_name} {cred_id} ~~" + '\x1B[0m')
+            print('\x1B[1;20;46m' + f"~~ Skipping {check.get('name')} As {cred_name} Credential is Not Active ~~" + '\x1B[0m')
```

Comment:
Remove this, as we are already capturing this in the summary table.

Reply:
Removed